examples/arithmetics/src/cli.ts

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import * as fs from 'fs';
+import colors from 'colors';
+import { Command } from 'commander';
+import { Grammar, LangiumDocumentConfiguration, DefaultDocumentValidator } from 'langium';
+import { createArithmeticsServices } from './language-server/arithmetics-module';
+import { isModule } from './language-server/generated/ast';
+import { ArithmeticsInterpreter } from './interpreter';
+
+const program = new Command();
+program
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    .version(require('../package.json').version);
+
+program
+    .command('eval')
+    .description('calculate Evaluations in the .calc file')
+    .option('-f, --file <file>', 'the .calc file')
+    .action((options: EvalOptions) => {
+        if (!options.file) {
+            console.log('Please, enter a file name.');
+            process.exit(1);
+        }
+        if (!isCalcExtension(options.file)) {
+            console.log('Please, choose a file with the .calc extension.');
+            process.exit(1);
+        }
+
+        if (!fs.existsSync(options.file)) {
+            console.log('File doesn\'t exist: ' + options.file);
+            process.exit(1);
+        }
+        const fileContent = fs.readFileSync(options.file).toString();
+
+        const arithmeticsServices = createArithmeticsServices();
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const languageId = require('../package.json').contributes.languages.id;
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const version = require('../package.json').version;
+        const document = LangiumDocumentConfiguration.create(`file:${options.file}`, languageId, version, fileContent);
```

Comment:
The `version` is not the package version, but the document version (in the LSP context the document version is increased after every change sent from the client). You can set it to `0`.